package.json

```diff
@@ -77,7 +79,7 @@
     "react-native-svg": "12.1.1",
     "react-native-vector-icons": "9.0.0",
     "react-native-video": "5.2.0",
-    "react-native-webrtc2": "1.84.5",
+    "react-native-webrtc": "github:streamer45/react-native-webrtc",
```

Comment:
are we going to keep the fork? if we are, should we consider moving it under the `mattermost` organization?

Reply:
That's TBD at the moment but if it becomes a permanent thing I agree we should move it there.